CSS/components/alert.css

```diff
@@ -0,0 +1,28 @@
+.avatar{
+    border-radius: 100%;
+    width: 5rem;
+    height: auto;
+    cursor: pointer;
+}
+.avatar-l{
+    width: 6rem;
+    height: auto;
```

Comment:
give height same value as width to make a circle

Reply:
Thank you Tahir for your suggestions, Made all the changes.
![Screenshot (404)](https://user-images.githubusercontent.com/75560307/152094903-2e8ae4da-5e49-4bbc-8521-250bdefb7b4a.png)